File: src/Counsel/Player.swift
//
//  Player.swift
//  Counsel
//
//  Created by Han Hossain on 9/29/19.
//  Copyright © 2019 Han Hossain. All rights reserved.
//

import Foundation

struct Player {
    let firstName: String
    let lastName: String
    let position: String
    let active: Bool
    let id: String
    let team: String?
    
    var fullName: String {
        return "\(firstName) \(lastName)"
    }
}

File: src/Counsel.Core/Models/Player.cs
namespace Counsel.Core.Models
{
	public class Player
	{
		public string Id { get; set; }

		public string FirstName { get; set; }

		public string LastName { get; set; }

		public string FullName => $"{FirstName} {LastName}";

		public string Position { get; set; }

		public string Team { get; set; }
	}
}
